fdbserver/storageserver.actor.cpp

```diff
@@ -2145,12 +2165,17 @@ bool expandMutation(MutationRef& m,
 void applyMutation(StorageServer* self, MutationRef const& m, Arena& arena, StorageServer::VersionedData& data) {
 	// m is expected to be in arena already
 	// Clear split keys are added to arena
+	int mutationBytes = mvccStorageBytes(m) / 2;
+	self->updateMutationBytes += mutationBytes;
```

Comment:
The naming here is potentially a bit confusing. On the storage servers, we have two different concepts for writes:

`bytesInput` tracks the data coming into the storage server queue, and is counted based on the size as it would appear in the queue (`mvccStorageBytes`).

`mutationBytes` tracks the size of the mutations without MVCC accounting. 

I'm not sure specifically which one is desired for tracking here, though at the moment this is named like `mutationBytes` and has accounting more like `bytesInput`. My recommendation would be to use naming that matches the existing accounting, depending on which accounting method is chosen.

Reply:
+1 we should have `fetchKeysMutationBytes` and `fetchKeysBytesInput` to be directly comparable to the existing storage metrics.

Reply:
Good point. I checked the metrics in `counters` and left more comments. Because fetch keys go to storage directly, they are not 100% comparable. So I'm also adding the number of KVs.